### PR TITLE
chore: [IOPID-2163] Delete local FF to activate new scan and settings sections

### PR DESCRIPTION
**Pull request by Ladirico**
## Short description
Delete local FF and switch, in order to activate new scan and settings sections. Now the logic to show this new sections based on `newPaymentSection` remote FF. 

### 🤖 Android 🤖

<p>

| Production | First Onboarding dev-server | already logged dev-server |
| - | - | - |
| <video src="https://github.com/user-attachments/assets/c6904b20-01c1-442d-a584-4ba987d365e8"/> | <video src="https://github.com/user-attachments/assets/f1e56241-310d-40a3-96b1-967038dd6ce0"/> | <video src="https://github.com/user-attachments/assets/896c1531-4526-4c23-8f25-f4ec0e371926"/> | 

</p>

### 🍏 iOS 🍏

<p>

| Production | Onboarding dev-server | already logged dev-server |
| - | - | - |
| <video src="https://github.com/user-attachments/assets/72c53525-1e80-40ea-92d4-bc1dd19fc691"/> | <video src="https://github.com/user-attachments/assets/01396d37-2a5d-497c-9b92-dc8c6ccf541a"/> | <video src="https://github.com/user-attachments/assets/1e1487b5-8a9e-4798-a46f-d39679197062"/> | 

</p>

## How to test
If you're using the io-dev-api-server (local environment) need to change [this remote FF](https://github.com/pagopa/io-dev-api-server/blob/cc4e3ee44a9ae22bd04fd970b43a621f71e04f23/src/payloads/backend.ts#L106) using this configuration:
 ```
newPaymentSection: {
      enabled: false,
      min_app_version: {
        ios: "1.0.0",
        android: "1.0.0"
      }
    }
```
After, you can log into the app.
Otherwise, if you're using the production environment you only need to log into the app.
Once you are logged into the app in the centre of the tab bar, you should have the ‘scan’ button and no longer have the Profile section in the tab bar, but the settings icon should be visible in each of the top-level screens allowing you to access the settings.

### CODE REVIEW

**Review by shadowsheep1 — APPROVED** [review]
LGTM

File: ts/store/reducers/persistedPreferences.ts [hunk 1]
```diff
@@ -59,8 +57,7 @@ export const initialPreferencesState: PersistedPreferencesState = {
   isMixpanelEnabled: null,
   isPnTestEnabled: false,
   isIdPayTestEnabled: false,
-  isDesignSystemEnabled: false,
-  isNewScanSectionEnabled: false
+  isDesignSystemEnabled: false
```
[the remark this answers is not in the document]
Comment (Ladirico) [reply ts/store/reducers/persistedPreferences.ts:60]: Hi @LazyAfternoons  Thank you for the suggestion, but we had already considered this and decided not to add the migration of the persistent state both when adding data to the state and when removing data from the state. We do not expect the data to be reinserted later, so we do not expect any conflicts.

[the remark this answers is not in the document]
Comment (LazyAfternoons) [reply ts/store/reducers/persistedPreferences.ts:60]: I'm all for being explicit with migrations but I'm okay with moving forward as is, thanks

**Review by ChrisMattew — APPROVED** [review]

File: ts/store/reducers/persistedPreferences.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (Ladirico) [reply ts/store/reducers/persistedPreferences.ts:60]: In the end, your comment made me reconsider the choice. I just made the commit that adds the migration.
Thanks a lot @LazyAfternoons

**Review by shadowsheep1 — APPROVED** [review]
LGTM 
```zsh
 LOG  redux-persist: running migration for versionKey 34
```

**Pull request merged**